### PR TITLE
Add instructions for Anaconda installation to README

### PR DESCRIPTION
**Pull request by kynan**

### CODE REVIEW

**Comment by justin1121** [comment]
I think we should just link to the tutorials here https://github.com/OpenMined/tutorials

File: README.md [hunk 1]
```diff
@@ -16,16 +16,27 @@ The goal of this library is to give the user the ability to efficiently train De
...
 
-### Prerequisites
+### Using [Anaconda](http://anaconda.org) (recommended)
+
+- [Install Anaconda](https://www.anaconda.com/download/) or [Miniconda](https://conda.io/miniconda.html) for your platform
+- Install our OpenMined Unity app by following the guidelines [here](https://github.com/OpenMined/OpenMined)
+- Install PySyft
+  - Linux: run `./install_for_anaconda.sh`
+  - Windows: execute `https://conda.io/miniconda.html`
+
```
Comment (justin1121) [inline README.md]: Link to tutorials instead, I think. https://github.com/OpenMined/tutorails

Comment (bendecoste) [reply README.md]: your link is broken! https://github.com/OpenMined/tutorials

Comment (justin1121) [reply README.md]: Whoops!

**Comment by kynan** [comment]
The tutorials are deprecated, so I think these instructions are still useful, no?

**Pull request closed without merging**